src/schema/borough.ts

```diff
@@ -0,0 +1,10 @@
+import { InferSelectModel } from "drizzle-orm";
+import { char, pgTable, text } from "drizzle-orm/pg-core";
+
+export const borough = pgTable("borough", {
+  id: char("id", { length: 1 }).primaryKey(),
+  title: text("title").notNull(),
+  abbr: text("abbr").notNull(),
+});
+
+export type SelectBorough = InferSelectModel<typeof borough>;
```

Comment:
Any objection to just calling this type `Borough`? The drizzle docs [seem to use that pattern too](https://orm.drizzle.team/docs/sql-schema-declaration), though I also vaguely remember seeing this pattern in there docs somewhere.

Reply:
Calling it 'Borough' makes sense. I had only seen the 'Select[Schema]' and 'Insert[Schema]' pattern and it was rubbing me the wrong way. The '[Schema]' and `[NewSchema]` convention strikes me as more concise and more inline with JS conventions.